page/infrastruktur-perhubungan.php

```diff
@@ -0,0 +1,73 @@
+            <p>Pada Januari 2008, pengembangan dilakukan kembali dengan memproduksi alat penambat rel type e-Clip kemudian dilakukan test track. Hingga pada tahun 2010 mendapat izin penggunaan pengesahan dan rekomendasi pemakaian dari Direktur Jenderal Perkeretaapian Kementerian Perhubungan No. KA.405/SK.10/DJKA/II/10</p>
+            <p>Alat Penambat Rel (Rail fastener) produksi Pindad sangat kompetitif, dengan jaminan kualitas pekerjaan yang tinggi dimana dalam proses produksi kami menggunakan Standar Iinternational maupun National seperti AREMA/AREA Standard, SNI Standar, serta didukung oleh sumber daya manusia yang telah berpengalaman dengan kemampuan dan keahlian yang sudah teruji.</p>
+            <p>Produk Peralatan Kapal Laut bermula dari kerja sama yang didirikan bersama Hatlapa. Pindad telah mengembangkan berbagai produk peralatan kapal laut sejak tahun 1991 hingga saat ini dengan nama PinMarine. Dengan dukungan Sumber Daya Manusia, sistem produksi dan kualitas kontrol yang baik, PinMarine mampu menghasilkan produk berkualitas dan kompetitif, serta memenuhi persyaratan klasifikasi Nasional & Internasional, seperti BKI, LR, ClassNK, BV, RINA, GL, ABS, dan lain-lain.</p>
+            <p>Kualitas hasil produksi yang bermutu tinggi dan kepuasan pelanggan dalam menggunakan produk yang kami hasilkan. merupakan tujuan bisnis Pindad. Dengan semangat dan keyakinan yang tinggi kami selalu siap bekerja keras untuk mengembangkan keahlian dan teknologi yang dimiliki agar dapat menawarkan solusi terbaik bagi pelanggan maupun pengguna produk kami.</p>
+            <p>Untuk pemesanan produk, silahkan kunjungi link berikut ini : <a href="https://pindad.com/inquiry-process">https://pindad.com/inquiry-process</a></p>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-md-3">
+            <input type="text" class="search" name="search" placeholder="Quick Search">
+            <ul class="fa-ul list-aside my-3">
+                <li class="py-1"><a href="#">All</a></li>
+                <li class="py-1"><a href="#">Forging</a></li>
+                <li class="py-1"><a href="#">Casting</a></li>
+                <li class="py-1"><a href="#">Railway Equipment</a></li>
+                <li class="py-1"><a href="#">Ship Equipment</a></li>
+            </ul>
+        </div>
+        <div class="col-md-9 row">
+            <?php
+                $select = mysqli_query($connection, "select * from produk where jenis_produk='Infrastruktur Perhubungan'");
+                while($data = mysqli_fetch_array($select)){
+                    echo "<div class='col-md-3 mb-4'>";
+                    echo "<div class='card'>";
+                    echo "<img class='card-img-top' src='admin/upload/".explode("-", $data["gambar"])[0]."/gambar/".explode("-", $data["gambar"])[1]."'>";
+                    echo "<div class='card-body font-weight-bold'>";
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)